qa-pipelines/tasks/run-test.sh

```diff
@@ -23,7 +23,9 @@ fi
 set -o nounset
 set -o allexport
 # Set this to skip a test, e.g. 011
-EXCLUDE_BRAINS_PREFIX=''
+EXCLUDE_BRAINS_PREFIX=
+# Set this to run only one test. If EXCLUDE and INCLUDE are both specified, EXCLUDE is applied after INCLUDE
+INCLUDE_BRAINS_PREFIX=${INCLUDE_BRAINS_PREFIX:-}
```

Comment:
Can we be more clear in comments about the expected/acceptable values for EXCLUDE_BRAINS_PREFIX and INCLUDE_BRAINS_PREFIX?